src/vsphere_cpi/spec/unit/cloud/vsphere/cluster_picker_spec.rb

```diff
@@ -85,9 +97,9 @@ def fake_datastore(name, free_space, mob = nil)
             picker = ClusterPicker.new(0, 0)
             picker.update(available_clusters)
 
-            expect do
+            expect {
               picker.best_cluster_placement(req_memory: 1024, disk_configurations: disks)
-            end.to raise_error(/No valid placement found/)
+            }.to raise_error(/No valid placement found/)
```

Comment:
What difference does it cause to use { vs do

Reply:
nothing, I just did it for consistency in this spec file